antelope/README.md

```diff
@@ -0,0 +1,27 @@
+---
+namespace-identifier: antelope
+title: Antelope Namespace
```

Comment:
```suggestion
title: Antelope and EOS Namespace
```
or
```suggestion
title: Antelope (formerly known as EOS) Namespace
```
or something else that telescopes to unfamiliar readers the scope

Reply:
I followed your second suggestion